src/helpers.js

```diff
@@ -33,3 +33,8 @@ var toggleProp = function(defaultState, alternateState) {
 
   return p;
 };
+
+//Templates
+var loader = function(){
+  return m("img[alt='Loader'][src='/assets/catarse_bootstrap/loader-eff2ad1eeb09a19c9afb5b143e1dd62b.gif']");
```

Comment:
we should avoid using the assets from rails assets pipeline like this. The hash used won't be synced automatically. The ideal is using a loader in our public folder or directly form S3.